modules/contentbox/models/system/CBHelper.cfc

```diff
@@ -201,7 +201,7 @@ component accessors="true" singleton threadSafe{
 	function siteBaseURL(){
 		return replacenocase( 
 			getRequestContext().buildLink( linkTo='', ssl=getRequestContext().isSSL() ), 
-			"index.cfm", 
+			"index.cfm/", 

```

Comment:
Remember, its removing the `/`
By default it shows as `http://127.0.0.1:8080/index.cfm/`
So the base url replaces it and becomes `http://127.0.0.1//`